src/core/src/lib/components/formly.attributes.ts

```diff
@@ -135,11 +132,18 @@ export class FormlyAttributes implements OnChanges, DoCheck, OnDestroy {
   }
 
   private attachAttrElement() {
-    this.fieldAttrElements = [...this.fieldAttrElements, this.elementRef.nativeElement];
+    if (!this.field['_attrElements']) {
```

Comment:
instead of using `!` here, just reverse your if else statement

Reply:
I don't see the diff, for readability purpose I would prefer my approach (init first and push after)

Reply:
there is no difference, the only thing is you are doing an extra operation, even though the operation is not expensive, but when it can be avoided just by simply changing the order of if else, then why should we add extra operation

Reply:
maybe I'm wrong but still don't see the extra operation, `if-else` is one condition similar to `?:` operator, anyway I'll make those changes in my next PR to make everyone happy, thanks!